changelog.d/1778

```diff
@@ -0,0 +1,8 @@
+synopsis: Remove GHC <= 8.10.7 from the support window
+prs: #1778
+
+description {
+  GHC <= 8.10.7 is removed from the support window. The lowest version supported in CI becomes GHC 9.2.1, released on the 29/10/2021.
```

Comment:
If the lower base bound is `4.16.4.0`, then that corresponds to 9.2.5, not 9.2.1. So I think this should say 9.2.5

Reply:
you're right, actually latest patch version of 9.2 (9.2.8) should be the one we should aim